pom.xml

```diff
@@ -124,6 +124,7 @@
     <jackson.bundle.version>${jackson.version}</jackson.bundle.version>
     <slf4j.version>2.0.16</slf4j.version>
     <log4j.version>2.24.0</log4j.version>
+    <logback.version>1.3.14</logback.version>
```

Comment:
I introduced logback here. Hope you are fine with it.
We need it to set dynamically the log levels.

If you don't like this it should be easy to switch to log4j.

Reply:
I think it might be better if we stick to log4j better simply rely on slf4j if possible.